Explorer/Assets/DCL/UI/ColorPicker/ColorPickerController.cs

```diff
@@ -38,6 +39,7 @@ public ColorPickerController(ColorPickerView view, ColorToggleView colorToggle,
                 actionOnRelease:(toggle) => toggle.gameObject.SetActive(false));
 
             view.SliderHue.Slider.onValueChanged.AddListener(_ => SetColor());
+            view.SliderHue.Slider.onValueChanged.AddListener(_ => SetSaturationColor());
```

Comment:
Can we check why these listeners arent removed?

Reply:
It was a miss. I've added the IDisposable interfaces on all controllers that needed it and called the Dispose() function where needed